sql/python/sqlflow_submitter/db.py

```diff
@@ -16,13 +16,14 @@
 
 def connect(driver, database, user, password, host, port):
     if driver == "mysql":
-        from mysql.connector import connect
+        # NOTE: use MySQLdb to avoid bugs like infinite reading:
```

Comment:
Add a unit test for the infinite bug or merge https://github.com/sql-machine-learning/sqlflow/pull/611 firstly?

Reply:
Done.